tck/README.md

```diff
@@ -479,6 +479,22 @@ public class SkippingIdentityProcessorTest extends IdentityProcessorVerification
 }
 ```
 
+## Which verifications must be implemented by an compliant implementation?
+In order to be considered an Reactive Streams compliant we require implementations to cover their
```

Comment:
Can it be made even simpler? Something like: In order to be a Reactive Streams implementation, aside from following the specification, one must also make sure that the implementations of the Publisher and Subscriber pass the TCK?